uploader.json

```diff
@@ -1330,6 +1330,11 @@
             ],
             "symbolic_name": "cinder_volume_log"
         },
+        {
+            "file": "/var/log/mistral/executor.log",
+            "pattern": [],
+            "symbolic_name": "mistral_executor_log"
```

Comment:
@akshay196 we try to keep the `symbolic_name` in alpha sorted order, would you mind to move it to the `m's` section?